spec/requests/civil_servants/civil_servants_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CivilServants::CivilServantsController, type: :request do
+  describe '#edit' do
+    let(:perform_request) { get edit_civil_servants_civil_servant_path }
+
+    context 'when a civil servant is signed in' do
+      let(:civil_servant) { create :civil_servant, :full }
+
+      before { sign_in civil_servant.user }
+
+      it 'returns http success' do
+        perform_request
+        expect(response).to have_http_status(:success)
+      end
+    end
+  end
+
+  describe '#update' do
+    let(:perform_request) do
+      patch civil_servants_civil_servant_path(civil_servant, params: request_params, format: :js)
+    end
+    let(:civil_servant) { create :civil_servant, :full }
+    let(:request_params) do
+      {
+        civil_servant: update_params,
+        form_partial: 'personal_information_form'
+      }
+    end
+    let(:update_params) do
+      {
+        first_name: 'Lara',
+        last_name: 'Croft',
```

Comment:
Lol, this spec data seems quite familiar to me…

Reply:
Hm I don't know why that could be